sandstorm.js

```diff
@@ -109,7 +109,7 @@ if (isSandstorm && Meteor.isClient) {
   // sandstorm client to return relative paths instead of absolutes.
   const _absoluteUrl = Meteor.absoluteUrl;
   const _defaultOptions = Meteor.absoluteUrl.defaultOptions;
-  Meteor.absoluteUrl = (path, options) => {
+  Meteor.absoluteUrl = (path, options) => { // eslint-disable-line meteor/core
```

Comment:
Usually ESLint-plugin-Meteor warns when any part of the Core-API is overwritten. Since monkey-patching is used here, and we seem to know what we are doing, the rule is disabled :-)